pkgs/data/fonts/fira-math/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchurl, stdenv }:
+
+let
+  pname = "fira-math";
+  version = "0.3.4";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://github.com/firamath/firamath/releases/download/v${version}/FiraMath-Regular.otf";
+    sha256 = "sha256-ICjL091NjAzxYIUg60dZlWqDpnkx17bY58MTUgGG41s=";
+    name = "FiraMath-Regluar.otf";
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
+    install -D $src -t $out/share/fonts/opentype/
+  '';
```

Comment:
We can do this in only fetchurl and postFetch. see https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/fira/default.nix

Reply:
```nix
{ lib, fetchurl, stdenv }:

let
  version = "0.3.4";
in
fetchurl {
  name = "fira-math-${version}";

  url = "https://github.com/firamath/firamath/releases/download/v${version}/FiraMath-Regular.otf";

  postFetch = ''
    install -D $downloadedFile -t $out/share/fonts/opentype/
  '';

  sha256 = "sha256-ICjL091NjAzxYIUg60dZlWqDpnkx17bY58MTUgGG41s=";

  meta = with lib; {
    description = "Math font with Unicode math support based on FiraSans and FiraGO";
    homepage = "https://github.com/firamath/firamath";
    license = licenses.ofl;
    maintainers = [ maintainers.loicreynier ];
    platforms = platforms.all;
  };
}
```

While trying to do so I have an error at the `$out` directory creation that I can't solve.

Reply:
Looks correct to me. What is the error exactly?

Reply:
```
install: failed to access '/nix/store/9pq9bzkf22j889j41dhpb8rk3p7d88v5-fira-math-0.3.4/share/fonts/opentype/': Not a directory
```
or with `mkdir`

```
mkdir: cannot create directory '/nix/store/9pq9bzkf22j889j41dhpb8rk3p7d88v5-fira-math-0.3.4': File exists
```